Lanchonete/Lanchonete/ Features/Customization/Data/IngredientCellManager.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  IngredientCellManager.swift
+//  Lanchonete
+//
+//  Created by vera.dias on 2/10/21.
+//
+
+import Foundation
+
+public class IngredientCellManager {
+    var ingredientQuantity: Int = 0
+    
+    init(with quantity: Int? = nil) {
+        if let quantityFinal = quantity {
+            self.ingredientQuantity = quantityFinal
+        }
+        self.ingredientQuantity = 0
+    }
+    
+    func setQuantity(value: String) {
+        ingredientQuantity = Int (value) ?? 0
+    }
+    
+    func getQuantity() -> String {
+        return String (ingredientQuantity)
+    }
```

Comment:
This `Int <space> (stuff)` is a bit weird and confusing. Please avoid it as it looks like it's something different from an initializer call. Looks more like `if (value)`… 🙂

Reply:
Changed this also!